### ISSUE TITLE
Use Augmentor to predict a image, but don't know how to  pre-process test image

### ISSUE THREAD

**Issue by ideaRunner** [mdbloice/Augmentor#64]
Hi,
    Now, I am using Augmentor to get a generator and use Keras fit_generator function to train my model.
    After I trained, I am facing a problem.
    I don't know how Augmentor pre-processing the train set. But as a result it normalized the input image value between 0 and 1. So I don't know how to pre-process my test set so that it can be predict from the trained model.
    I tried to devide my test set by 255.. but the result is wrong...
    Can you give me any help? Or just show how you reappear your image processed by Augmentor. Then I will know how to go on.
    Thank you !

**Comment by mdbloice** [comment]
Hi there, yeah Augmentor is just doing something like:

```python
X = X.astype('float32')
X /= 255
```
so doing that to your test images should work when passing them through any trained model.

However, I am currently reworking all the generator functions, and the functions that work with images passed as arrays. I will soon push (next week) a new version of Augmentor to PyPI which will be more feature complete, so if you can wait a few days and use the newer version it may solve the issues you are currently having. 
Hope this helps,
M.

**Comment by ideaRunner** [comment]
Hi Mdbloice, 
Thank you for your reply. It really works!
Augmentor help me a lot to pre-processing image, it is a wonderful project.
I am looking forward  to use new version!
Thank you again.
